functions/src/index.ts

```diff
@@ -88,6 +88,9 @@ exports.calculateAqi = functions.pubsub
   .schedule('every 10 minutes')
   .onRun(async () => {
     const sensorList = (await firestore.collection('sensors').get()).docs;
+    const previousDataDoc = (
```

Comment:
Being another read, does this impact Amber's calculations?

Reply:
The next PR removes the call to the sensors collection (I had to add the readingDocId first to be able to do this), so it will reduce the number of reads every 2 minutes from O(num sensors) to O(1), which I'm not sure we were actually accounting for currently?